packaging/android/build.sh

```diff
@@ -165,7 +167,7 @@ fi
 if [ ! -e "$PKG_CONFIG_LIBDIR/sqlite3.pc" ] ; then
 	mkdir -p sqlite-build-"$ARCH"
 	pushd sqlite-build-"$ARCH"
-	../sqlite-autoconf-${SQLITE_VERSION}/configure --host=${BUILDCHAIN} --prefix="$PREFIX" --enable-static --disable-shared
+	../sqlite-autoconf-${SQLITE_VERSION}/configure --host=${BUILDCHAIN} --prefix="$PREFIX" --enable-static --disable-shared CFLAGS="-D__ANDROID_API__=$ANDROID_API"
```

Comment:
CFLAGS="-D__ANDROID_API__=$ANDROID_API"

Is this really the correct way of doing this in the ndk?

In ndk 15 this is handled by the wrapper shellscript it generates.